frontend/src/components/beta/transactions/TransactionHeader.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import { styled } from "../../../libraries/styles";
+import { shortenString } from "../../../libraries/formatting";
+import Moment from "../../../libraries/moment";
+
+import CopyToClipboard from "../../beta/common/CopyToClipboard";
+import { NearAmount } from "../../utils/NearAmount";
+import TransactionStatus from "./TransactionStatus";
+import { TransactionDetails } from "../../../types/common";
+import UtcLabel from "../common/UtcLabel";
+
+type Props = {
+  transaction: TransactionDetails;
+};
+
+const Wrapper = styled("div", {
+  display: "flex",
+  justifyContent: "space-between",
+  width: "100%",
+  backgroundColor: "#1b1d1f",
+  padding: 28,
+  borderRadius: 8,
+  fontFamily: "Manrope",
+});
+
+const BaseInfo = styled("div", {
+  display: "flex",
+});
+
+const AuthorInfo = styled("div", {
+  display: "flex",
+  alignItems: "center",
+});
+
+const Author = styled("div", {
+  display: "flex",
+  alignItems: "center",
+  color: "#fff",
+  fontWeight: 700,
+  fontSize: 30,
+  lineHeight: "150%",
+});
+const Avatar = styled("div", {
+  size: 26,
+  backgroundColor: "#c4c4c4",
+  opacity: 0.2,
+  borderRadius: "50%",
+  marginRight: 8,
+});
+
+const Divider = styled("img", {
+  width: 17,
+  marginHorizontal: 20,
+});
+
+const TransactionHash = styled("h1", {
+  fontFamily: "SF Pro Display",
+  fontSize: 18,
+  fontWeight: 400,
+  color: "#fff",
+  lineHeight: "21px",
+  marginRight: 14,
+  marginLeft: 34,
+  marginBottom: 0,
+});
+
+const TransactionCopy = styled("span", {
+  marginLeft: ".5em",
+});
+
+const BaseInfoDetails = styled("div", {
+  display: "flex",
+  alignItems: "center",
+  marginTop: 24,
+});
+
+const NumericInfo = styled("div", {
+  display: "flex",
+  alignItems: "center",
+});
+const NumericDivider = styled("div", {
+  height: "100%",
+  width: 1,
+  marginHorizontal: 20,
+});
+
+const AmountHeader = styled("div", {
+  fontSize: 12,
+  color: "#c9c9c9",
+  lineHeight: "14px",
+  fontFamily: "SF Pro Display",
+});
+
+const Amount = styled("div", {
+  fontWeight: 500,
+  fontSize: 20,
+  color: "#fff",
+  marginTop: 16,
+  fontFamily: "SF Mono",
+});
+
+const TimeWrapper = styled(AuthorInfo);
```

Comment:
Do we need a wrapper in this case?
Should we name it `CenteredContainer` in this case?